src/Commands/FullIntake.cpp

```diff
@@ -0,0 +1,14 @@
+#include "FullIntake.h"
+#include "SetArms.h"
+#include "IntakeOn.h"
+#include "ShooterControl.h"
+#include "RollerOn.h"
+
+FullIntake::FullIntake(double speed)
+: mSpeed(speed)
+{
+  AddSequential(new SetArms(Intake::kDown));
```

Comment:
Good observation. We use `Intake::Down` which is in `Subsystems/Intake.h`. However, the reason we are using `Intake:Down` is because we are constructing the command `SetArms`. `SetArms` requires that we use `Intake:Down`. `SetArms` will define what is required in order to use that interface.

So, no change required as `SetArms.h` is already included.